backend/src/main/java/ai/verta/modeldb/versioning/blob/container/EnvironmentContainer.java

```diff
@@ -227,23 +222,9 @@ private String computeSHA(PythonEnvironmentBlob blob) throws NoSuchAlgorithmExce
     sb.append("python:");
     appendVersion(sb, version);
     sb.append(":requirements:");
-    for (PythonRequirementEnvironmentBlob pythonRequirementEnvironmentBlob :
-        blob.getRequirementsList()) {
-      sb.append(":library:")
-          .append(pythonRequirementEnvironmentBlob.getLibrary())
-          .append(":constraint:")
-          .append(pythonRequirementEnvironmentBlob.getConstraint());
-      appendVersion(sb, pythonRequirementEnvironmentBlob.getVersion());
-    }
+    sb.append(new HashSet<>(blob.getRequirementsList()).hashCode());
```

Comment:
would hahsCode be same across calls/ across restarts?
we would want the same content of requirementList to be create identical sha

Reply:
changed base and used generated files to compute hash